README.md

```diff
@@ -254,46 +254,44 @@ have a suggestion for another dataset to include in PyKEEN, please let us know
-| True Negative Rate                          | `[0,1]`    | 📈          | https://en.wikipedia.org/wiki/Specificity_(tests)                                               | Classification |
-| True Positive Rate                          | `[0,1]`    | 📈          | https://en.wikipedia.org/wiki/Sensitivity_(test)                                                | Classification |
-| Adjusted Arithmetic Mean Rank (AAMR)        | `(0, 2)`   | 📉          | The mean over all chance-adjusted ranks.                                                        | Ranking        |
-| Adjusted Arithmetic Mean Rank Index (AAMRI) | `[-1, 1]`  | 📈          | The re-indexed adjusted mean rank (AAMR)                                                        | Ranking        |
-| Geometric Mean Rank (GMR)                   | `[1, inf)` | 📉          | The geometric mean over all ranks.                                                              | Ranking        |
-| Harmonic Mean Rank (HMR)                    | `[1, inf)` | 📉          | The harmonic mean over all ranks.                                                               | Ranking        |
-| Hits @ K                                    | `[0, 1]`   | 📈          | The relative frequency of ranks not larger than a given k.                                      | Ranking        |
-| Inverse Arithmetic Mean Rank (IAMR)         | `(0, 1]`   | 📈          | The inverse of the arithmetic mean over all ranks.                                              | Ranking        |
-| Inverse Geometric Mean Rank (IGMR)          | `(0, 1]`   | 📈          | The inverse of the geometric mean over all ranks.                                               | Ranking        |
-| Inverse Median Rank                         | `(0, 1]`   | 📈          | The inverse of the median over all ranks.                                                       | Ranking        |
-| Mean Rank (MR)                              | `[1, inf)` | 📉          | The arithmetic mean over all ranks.                                                             | Ranking        |
-| Mean Reciprocal Rank (MRR)                  | `(0, 1]`   | 📈          | The inverse of the harmonic mean over all ranks.                                                | Ranking        |
-| Median Rank                                 | `[1, inf)` | 📉          | The median over all ranks.                                                                      | Ranking        |
+### Metrics (38)
+
+| Name                                        | Range    | Direction   | Description                                                 | Type           |
+|---------------------------------------------|----------|-------------|-------------------------------------------------------------|----------------|
+| AUC-PR                                      | [0,1]    | 📈          | Area Under the Precision-Recall Curve                       | Classification |
```

Comment:
suggestion to include the hyperlinks (if available):
```
[AUC-PR](https://en.wikipedia.org/wiki/Receiver_operating_characteristic)
```

Reply:
done for all metrics